tests/backends/test_arviz.py

```diff
@@ -630,15 +630,30 @@ def test_issue_5043_autoconvert_coord_values(self):
             # We're not automatically converting things other than tuple,
             # so advanced use cases remain supported at the InferenceData level.
             # They just can't be used in the model construction already.
-            converter = InferenceDataConverter(
-                trace=mtrace,
-                coords={
-                    "city": pd.MultiIndex.from_tuples(
-                        [("Bonn", 53111), ("Berlin", 10178)], names=["name", "zipcode"]
-                    )
-                },
-            )
-            assert isinstance(converter.coords["city"], pd.MultiIndex)
+            # TODO: we now ensure that everything passed to arviz is converted to
```

Comment:
Flagging this for discussion. This PR currently updates InferenceDataConverter to always convert coordinate values to ndarray. (Previously they were only converted if the sequence was a tuple.) Is there a reason we would want to maintain the original behavior...?

This test was originally made for #5043